test/Interop/Cxx/class/loadable-types-silgen.swift

```diff
@@ -0,0 +1,93 @@
+// RUN: %target-swift-emit-silgen -I %S/Inputs -enable-cxx-interop %s | %FileCheck %s
+
+import LoadableTypes
+
+// CHECK-LABEL: sil hidden [ossa] @$s4main21passAddressOnlyStruct{{[_0-9a-zA-Z]*}}F
+func passAddressOnlyStruct(s: EmptyStruct) {
+  // CHECK: bb0(%0 : $EmptyStruct):
+}
+
+// CHECK-LABEL: sil hidden [ossa] @$s4main32passStructWithDefaultConstructor{{[_0-9a-zA-Z]*}}F
+func passStructWithDefaultConstructor(s: StructWithDefaultConstructor) {
```

Comment:
You could reduce duplication in function names by calling all functions just `pass`. You will still find the type of the argument in the mangled name, so it would be possible to find the `CHECK-LABEL` line that uniquely identifies a function.

Reply:
What you said was not quite true, because some structs mangle differently. For example `EmptyStruct` get's mangled into something that doesn't contain that name. I simplified it in a different way.

Reply:
Correction: I fell victim to Swift's compression in mangling. I'd still keep it like it is.